hadoop-common-project/hadoop-common/src/main/java/org/apache/hadoop/security/ShellBasedUnixGroupsMapping.java

```diff
@@ -215,7 +215,13 @@ private Set<String> getUnixGroups(String user) throws IOException {
           groups = resolvePartialGroupNames(user, e.getMessage(),
               executor.getOutput());
         } catch (PartialGroupNameException pge) {
-          LOG.warn("unable to return groups for user {}", user, pge);
+          if(LOG.isDebugEnabled()) {
```

Comment:
can we just extract this into String msg = ""unable to return groups for user {}" and use this msg to make log line looks clean?

Reply:
@smengcl do you think this is fine? or we don't need this in even in Warn? 
we were hitting this log message. This current patch voids the full trace in logs when debug is not enabled.

Reply:
@swamirishi could you please fix the @smengcl comments above? you need to format line number 218

Reply:
I see this is stale version. Thanks